packages/file-type-icons/src/FileTypeIconMap.ts

```diff
@@ -28,7 +28,6 @@ export const FileTypeIconMap: { [key: string]: { extensions?: string[] } } = {
       'mid',
       'mmf',
       'mp3',
-      'ogg',
```

Comment:
Is this actually a dupe?

Reply:
yes, it was listed in both the video and audio sections. ogg audio files are oga, this was an oversight in older versions of the file.